Editor/AGS.Editor/Components/AudioComponent.cs

```diff
@@ -323,25 +325,67 @@ private void ImportAudioFiles(string[] selectedFiles)
             }
         }
 
+        private AudioClip SetAudioClipSourceFile(AudioClip clip, string fullSourceFileName)
+        {
+            if (string.IsNullOrEmpty(fullSourceFileName))
+            {
+                return null;
+            }
+
+            if(!File.Exists(fullSourceFileName))
+            {
+                throw new FileNotFoundException("Source file doesn't exist.");
+            }
+            else
+            {
+                byte[] audioData = File.ReadAllBytes(fullSourceFileName);
+                if (audioData.Length == 0) throw new FileLoadException("New source file is empty.");

```

Comment:
Why do you think this test is necessary, and is there other way to test other than loading whole file into memory?